### PR TITLE
Expose 'started_at' field in event api

### PR DESCRIPTION
**Pull request by bradur**
When using the API, the "display_dates" field format is inconsistent so it is hard to parse to an actual date. This PR will expose the "started_at" field which is a Date so it is easy to parse.

### CODE REVIEW

**Comment by mkalam-alami** [comment]
Looks good to me!

**Comment by mkalam-alami** [comment]
Up

**Comment by bradur** [comment]
Thanks, looking good!

`{
  "id": 4,
  "name": "1st-alakajam",
  "title": "1st Alakajam!",
  "started_at": "2017-07-10T00:00:00.000Z"
}`
